<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>9</y>
      <width>1031</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="approximationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Approximation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="linearInterpolationCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>115</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear interpolation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="quadraticInterpolationCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>133</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quadratic interpolation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lagrangeInterpolationCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagrange</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bezierCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezier</string>
    </property>
   </widget>
   <widget class="QLabel" name="interpolationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interpolation</string>
    </property>
   </widget>
   <widget class="QLabel" name="NLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N =</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="approximateCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Approximate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPowersButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add powers</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>158</width>
      <height>179</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>156</width>
       <height>177</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
